tests/thumbnails.ml

```diff
@@ -0,0 +1,53 @@
+open Lwt
+module D = Dropbox_lwt_unix
+
+(** If there is two entries, we call thumbnails with the path as
+       the first arg and the param format as the second.
+    If there is three entries, we add the param size as the third argument *)
+
+let string_to_size = function
+  | "xs" -> `Xs
+  | "s" -> `S
+  | "m" -> `M
+  | "l" -> `L
+  | "xl" -> `Xl
+  | _ -> invalid_arg "Size must be xs, s, m, l or xl."
+
+let string_to_format = function
+  | "jpeg" -> `Jpeg
+  | "jpg" -> `Jpeg
+  | "png" -> `Png
+  | "bmp" -> `Bmp
+  | _ -> invalid_arg "Format must be jpeg, jpg, png or bmp."
+
+
+let download t ?(format="jpeg") ?(size="s") fn =
+  D.thumbnails t ~size:(string_to_size size)
+               ~format:(string_to_format format) fn
+  >>= function
+    | None -> Lwt_io.printlf "No image named %S." fn
+    | Some(metadata, stream) ->
+  (* Save stream to the disk *)
+  let fn = Filename.basename fn in
+  let fname = "thumbnail_of_" ^ (Filename.chop_extension fn) ^ "." ^ format in
```

Comment:
First, i did a pattern matching for Filename.chop_extension in case of the file doesn't have an extension. But it seems that dropbox don't except such files to be used by thumbnails function.